pycytominer/aggregate.py

```diff
@@ -14,6 +14,8 @@
 def aggregate(
     population_df,
     strata=["Metadata_Plate", "Metadata_Well"],
+    compute_object_count=False,
```

Comment:
Can you move these to between `output_file` and `subset_data_df`? I think that the argument ordering should follow in order of how often a user interacts with it. In my mind:

Top args - most changed
output_file - the last argument in the top args (it has the biggest effect)
output_file args - any args pertaining to how the file is handled

also, do I even use `subset_data_df` in this function? 😬  If not, can you delete it?

Reply:
> Can you move these to between `output_file` and `subset_data_df`? 

Done

> also, do I even use subset_data_df in this function? 😬 If not, can you delete it? 

You do :) So I have left it as is.